lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -901,6 +901,11 @@ NativeCodeGenerator::CodeGen(PageAllocator * pageAllocator, CodeGenWorkItem* wor
             workItem->GetJITData(),
             scriptContext->GetRemoteScriptAddr(),
             &jitWriteData);
+        if (hr == E_ACCESSDENIED && body->GetScriptContext()->IsClosed())
```

Comment:
Do we need to restrict this with hr == E_ACCESSDENIED? Seems like what ever the error code is if SC is closed it should be aborted. Today the error code might be E_ACCESSDENIED but in future if any new error code is added this will break?